pkg/cloudprovider/provider/kubevirt/provider.go

```diff
@@ -1042,3 +1062,37 @@ func getStorageTopologies(ctx context.Context, storageClasName string, c *Config
 
 	return nil
 }
+
+func setOVNAnnotations(c *Config, annotations map[string]string) error {
```

Comment:
Would be cool to see effect of this function materialized in the testdata.

Reply:
Has been added